types/index.d.ts

```diff
@@ -52,12 +52,12 @@ interface BaseOptions<T extends JSToPostgresTypeMap> {
   debug: boolean | ((connection: number, query: string, parameters: any[]) => void);
   /** Transform hooks */
   transform: {
-    /** Transforms incoming column names */
-    column?: (column: string) => string;
-    /** Transforms incoming row values */
-    value?: (value: any) => any;
-    /** Transforms entire rows */
-    row?: (row: postgres.Row) => any;
+    /** Transforms incoming and outgoing column names. */
+    column?: (column: string) => string | { from?: (column: string) => string, to?: (column: string) => string };
+    /** Transforms incoming and outgoing row values. */
+    value?: (value: any) => any | { from?: (value: any) => any, to?: (value: any) => any };
+    /** Transforms entire incoming and outgoing rows. */
+    row?: (row: postgres.Row) => any | { from?: (row: postgres.Row) => any, to?: (row: postgres.Row) => any };
```

Comment:
@porsager is this the right signature for `to`?

Reply:
I assumes `from`'s output should be `to`'s input and vice versa, then I would suggest `{ from?: (row: postgres.Row) => any, to?: (row: any) => postgres.Row }`, but I'm not sure where `row.to` could be used...

Also, I guess `( )` are missing in the union? TS sees `(row: postgres.Row) => (any | { from?: (row: postgres.Row) => any, to?: (row: any) => any })`

```suggestion
    row?: ((row: postgres.Row) => any) | { from?: (row: postgres.Row) => any, to?: (row: any) => any };
```

Reply:
@Minigugus nicely spotted, thank you! Yes, I had the same feeling about the row. I updated my commit with your suggestions to avoid these micro-changes. Please feel free to review further.